source/plugin/workspace/netbeans.rst

```diff
@@ -1,80 +0,0 @@
-===================
-Setting Up NetBeans
```

Comment:
Because we don't have anyone familiar with netbeans to write the doc. If you're volunteering, go right ahead - but it would appear that there are *very few* sponge devs still using netbeans. We think it's better to drop it until / unless we have proper docs. For now, the old 3.0.0 docs are the best we have for netbeans support.